File: languages/python/bloock/entity/identity/attribute_descriptor.py
class AttributeDescriptor:
    """
    Represents a descriptor for an attribute.
    """
    def __init__(
            self, display_name: str, technical_name: str, description: str, required: bool
    ) -> None:
        """
        Constructs an AttributeDescriptor object with the specified parameters.
        :type required: object
        :type description: object
        :type technical_name: object
        :type display_name: object
        :rtype: object
        """
        self.display_name = display_name
        self.technical_name = technical_name
        self.description = description
        self.required = required

File: languages/php/Bloock/Entity/Identity/AttributeDescriptor.php
<?php

namespace Bloock\Entity\Identity;

/**
 * Represents a descriptor for an attribute.
 */
class AttributeDescriptor
{
    /**
     * Is the human-readable display name of the attribute.
     * @var string
     */
    public string $displayName;
    /**
     * Is the identifier for the attribute.
     * @var string
     */
    public string $technicalName;
    /**
     * Is a description providing additional information about the attribute.
     * @var string|null
     */
    public ?string $description;
    /**
     * Specifies whether the attribute is required.
     * @var bool
     */
    public bool $required;

    /**
     * @param string $displayName
     * @param string $technicalName
     * @param string|null $description
     * @param bool $required
     */
    public function __construct(string $displayName, string $technicalName, ?string $description, bool $required)
    {
        $this->displayName = $displayName;
        $this->technicalName = $technicalName;
        $this->description = $description;
        $this->required = $required;
    }


}